spec/serializers/institution_profile_serializer_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe InstitutionProfileSerializer, type: :serializer do
+  let(:institution) { build :institution }
+  let(:data) { JSON.parse(subject)['data'] }
+  let(:attributes) { data['attributes'] }
+  let(:links) { data['links'] }
+
+  subject { serialize(institution, serializer_class: described_class) }
+
+  it 'should include id' do
+    expect(data['id'].to_i).to eq(institution.id)
+  end
+
+  it 'links to vets api' do
+    expect(links['vet_website_link']).to_not be_empty
+  end
+
+  it 'should include OPE value' do
+    expect(data['attributes']['ope']).to eq(institution.ope)
+  end
```

Comment:
So from my reading on this the ope value (`object.ope` from `InstitutionProfileSerializer `) can be one of three things before it hits the serializer:

1) a non-nil value containing a "
2) a non-nil value that doesn't contain a "
3) a nil value

So to make sure this feature is working as intended, we'd need to ensure that this spec hits each of those variants.